app/views/courtSearchInfo.html

```diff
@@ -16,13 +16,13 @@ <h2 class="no-indent">Court Phone Number</h2>
 
         <div class="topMargin bottomMargin" ng-if="ctrl.courtWebsite">
           <h2 class="no-indent">Website</h2>
-          <p class="website no-indent" ng-bind-html="ctrl.courtWebsite | linky"></p>
+          <p class="website no-indent" ng-bind-html="ctrl.courtWebsite | linky:'_blank'"></p>
         </div>
 
         <div class="topMargin bottomMargin">
           <h2 class="no-indent">Your Legal Rights</h2>
           <p class="legalRights no-indent">
-            <a ng-click="ctrl.openLegalRightsLink()">{{ctrl.courtName}} Legal Rights</a>
+            <a ng-click="ctrl.openLegalRightsLink()" target="_blank">{{ctrl.courtName}} Legal Rights</a>
```

Comment:
This seems odd. `ctrl.openLegalRightsLink()` should be doing the opening of the page within that function using _blank. Because this isn't a regular HTML anchor tag link, I don't think the `target` does anything here.

Reply:
I tested this before submitting and it does work.  See: https://www.thesitewizard.com/html-tutorial/open-links-in-new-window-or-tab.shtml  as an example.